term.go

```diff
@@ -303,7 +319,7 @@ func (t *Terminal) Write(b []byte) (int, error) {
 }
 
 func (t *Terminal) setupShortcuts() {
-	t.ShortcutHandler.AddShortcut(&desktop.CustomShortcut{KeyName: fyne.KeyV, Modifier: fyne.KeyModifierShift | fyne.KeyModifierControl},
+	t.ShortcutHandler.AddShortcut(&desktop.CustomShortcut{KeyName: fyne.KeyV, Modifier: fyne.KeyModifierShortcutDefault},
```

Comment:
Oh sorry I was wrong and forgot about shift handling - this will use ctrl and command only - no shift.
As it's a terminal maybe it does need special handling because control needs shift but command does not.

Reply:
sorted :D